simulator/mips/t/unit_test.cpp

```diff
@@ -243,3 +206,454 @@ TEST_CASE( "MIPS32_instr_disasm: Process_Disasm_Branches")
     CHECK(MIPS32Instr(0x0c0004d2).get_disasm() == "jal 0x4d2");
 }
 
+
+// Converted SPIM TT tests
+// ************************* R INSTRUCTIONS ****************************
+// 19 done
+
+// Not implemented in SPIM TT
+TEST_CASE ( "MIPS32_instr: sll")
+{
+    MIPS32Instr instr( "sll");
+
+    instr.set_v_src( 1, 0);
+    instr.set_shamt( 0);
+    instr.execute();
+    CHECK( instr.get_v_dst() == 1);
+
```

Comment:
These should be different test cases.

Reply:
@pooh64 You still have one test case covering multiple execution patterns. We have to separate it to different test cases.

Reply:
sll, sra, srav are not implemented in SPIM TT. Should I modify that case(s), repeated in SPIM TT too?

Reply:
They are: https://github.com/MIPT-ILab/mips-traces/blob/master/tt.core32.s#L518

Their behavior is different for MIPS32 and MIPS64, we need to cover both cases.

Reply:
Now I'm splitting test cases. All written tests that have different behaviour for MIPS32 and MIPS64 currently are fully covered.